core/templates/pages/topics-and-skills-dashboard-page/topics-list/topics-list.directive.ts

```diff
@@ -85,11 +123,14 @@ angular.module('oppia').directive('topicsList', [
           };
 
           ctrl.$onInit = function() {
+            ctrl.getPageNumber = $scope.getPageNumber;
```

Comment:
Why are we using $scope and ctrl in this directive?

Reply:
Some properties are being passed from the parent controller and to avoid using them as $scope everywhere, I used ctrl for consistency.